truss/templates/control/proxy.conf

```diff
@@ -0,0 +1,28 @@
+
+server {
+    listen 8080;
+
+    # Liveness
+    location / {
+        add_header Content-Type text/plain;
+        return 200 "";
+    }
+
+    # Readiness
+    location ~ ^/v1/ {
+        if (!-f "~/inference_server_crashed.txt") {
+            return 503 '{"error": "Model load failed"}';
+        }
+
+        proxy_redirect off;
+        proxy_pass http://127.0.0.1:8090;
+    }
+
+    # Predict
+    location ~ ^/control {
+        proxy_redirect off;
+
+        proxy_pass http://127.0.0.1:8091;
```

Comment:
https://nginx.org/en/docs/http/ngx_http_upstream_module.html

Would be more idiomatic to use upstream module here and give you more flexibility over controlling endpoints

Reply:
```
upstream backend_v1 {
    server 127.0.0.1:8090;
}

upstream backend_control {
    server 127.0.0.1:8091;
}
```

something like this.

Reply:
```
proxy_pass http://backend_v1;
```

Reply:
Done!